integration/src/test/java/org/apache/iotdb/db/integration/IoTDBQueryWithIDTableIT.java

```diff
@@ -0,0 +1,291 @@
+        "insert into root.vehicle.d0(timestamp,s0) values(103,99)",
+        "insert into root.vehicle.d0(timestamp,s0) values(104,90)",
+        "insert into root.vehicle.d0(timestamp,s0) values(105,99)",
+        "insert into root.vehicle.d0(timestamp,s0) values(106,99)",
+        "insert into root.vehicle.d0(timestamp,s0) values(2,10000)",
+        "insert into root.vehicle.d0(timestamp,s0) values(50,10000)",
+        "insert into root.vehicle.d0(timestamp,s0) values(1000,22222)",
+        "insert into root.vehicle.d0(timestamp,s1) values(1,1101)",
+        "insert into root.vehicle.d0(timestamp,s1) values(2,198)",
+        "flush",
+        "insert into root.vehicle.d0(timestamp,s1) values(100,199)",
+        "insert into root.vehicle.d0(timestamp,s1) values(101,199)",
+        "insert into root.vehicle.d0(timestamp,s1) values(102,180)",
+        "insert into root.vehicle.d0(timestamp,s1) values(103,199)",
+        "insert into root.vehicle.d0(timestamp,s1) values(104,190)",
+        "insert into root.vehicle.d0(timestamp,s1) values(105,199)",
+        "insert into root.vehicle.d0(timestamp,s1) values(2,40000)",
+        "insert into root.vehicle.d0(timestamp,s1) values(50,50000)",
+        "insert into root.vehicle.d0(timestamp,s1) values(1000,55555)",
+        "insert into root.vehicle.d0(timestamp,s1) values(2000-01-01T08:00:00+08:00, 100)",
+        "insert into root.vehicle.d0(timestamp,s2) values(1000,55555)",
+        "insert into root.vehicle.d0(timestamp,s2) values(2,2.22)",
+        "insert into root.vehicle.d0(timestamp,s2) values(3,3.33)",
+        "insert into root.vehicle.d0(timestamp,s2) values(4,4.44)",
+        "insert into root.vehicle.d0(timestamp,s2) values(102,10.00)",
+        "insert into root.vehicle.d0(timestamp,s2) values(105,11.11)",
+        "insert into root.vehicle.d0(timestamp,s2) values(1000,1000.11)",
+        "insert into root.vehicle.d0(timestamp,s3) values(60,'aaaaa')",
+        "insert into root.vehicle.d0(timestamp,s3) values(70,'bbbbb')",
+        "insert into root.vehicle.d0(timestamp,s3) values(80,'ccccc')",
+        "insert into root.vehicle.d0(timestamp,s3) values(101,'ddddd')",
+        "insert into root.vehicle.d0(timestamp,s3) values(102,'fffff')",
+        "insert into root.vehicle.d0(timestamp,s3) values(2000-01-01T08:00:00+08:00, 'good')",
+        "insert into root.vehicle.d0(timestamp,s4) values(100, false)",
+        "insert into root.vehicle.d0(timestamp,s4) values(100, true)",
+        "insert into root.vehicle.d1(timestamp,s0) values(1,999)",
+        "insert into root.vehicle.d1(timestamp,s0) values(1000,888)",
+        "insert into root.other.d1(timestamp,s0) values(2, 3.14)",
+      };
+
+  private static boolean isEnableIDTable = false;
+
+  private static String originalDeviceIDTransformationMethod = null;
+
+  @BeforeClass
+  public static void setUp() throws Exception {
+    IoTDBDescriptor.getInstance().getConfig().setAutoCreateSchemaEnabled(true);
```

Comment:
By using new IT framework, altering config via IoTDBDescriptor is not worked in cluster test. Please add new methods to 'org.apache.iotdb.itbase.env.BaseConfig' and implements them.

Reply:
Thanks, this is not a cluster test, i will remove ClusterTest.class category

Reply:
OK, is this new feature worked in cluster, or only well done in standalone mode in current milestone?